### ISSUE TITLE
Initiële weergave van renvooi afhankelijk van status is ontwerp of definitief

### ISSUE THREAD

**Issue by HermanAssink** [IDgis/PlanoView2-bevindingen#223]
Als document status=ontwerp dan renvooi weergeven bij openen van het document, anders, als status=definitief dan renvooi niet weergeven bij openen van het document.

**Comment by AnkeVanBeek-IDgis** [comment]
@Ysbrand-OV : Is deze wens nog actueel, want er zijn andere use-cases waarbij dit gedrag onwenselijk is. En het kan conflicteren bij open van twee documenten, één ontwerp en één definitief.

**Comment by Ysbrand-OV** [comment]
@AnkeVanBeek-IDgis ja, die wens bestaat nog. Het gaat me om een default instelling (of een zelf te bepalen instelling). Dus niet dat renvooi niet mogelijk is bij vastgestelde plannen. Immers is het bij een vastgesteld (dus nu geldend) plan eigenlijk niet meer nodig om renvooi te tonen. De geïnteresseerde zou het zelf aan moeten kunnen zetten als ie toch het verschil zou willen zien. Bij een ontwerp is het juist wel nodig om de verschilweergave met de nu geldende versie te kunnen bekijken en dus handig als het default "aan" staat.

**Comment by AnkeVanBeek-IDgis** [comment]
Afhankelijk van het regelingtype en status (ontwerp/definitief) staat renvooi nu initiëel nu default aan of uit.
Bij een ontwerp omgevingsplan en omgevingsvisie van een gemeente staat bij een ontwerp renvooi nu default uit.
Bij ontwerp omgevingsvisies van provincie en rijk staat renvooi default aan evenals alle andere regelingtypen met status ontwerp.
Bij alle documenten met status definitief staat renvooi default uit.

In de admin kan de initiële instelling van renvooi aan/uit voor een document ingesteld worden. Deze instelling overruled de automatische initiële instelling zoals hiervoor beschreven.
